lib/bios/menus.robot

```diff
@@ -410,11 +410,21 @@ Reset To Defaults Tianocore
     [Documentation]    Resets all Tianocore options to defaults. It is invoked
     ...    by pressing F9 and confirming with 'y' when in option
     ...    setting menu.
+    Power On
+    ${main_menu}=    Enter Setup Menu Tianocore And Return Construction
     Read From Terminal
     Press Key N Times    1    ${F9}
     Read From Terminal Until    ignore.
     Write Bare Into Terminal    y
 
+    ${dasharo_menu}=    Enter Dasharo System Features    ${main_menu}
+    ${serial_menu}=    Enter Dasharo Submenu    ${dasharo_menu}    Serial Port Configuration
```

Comment:
I expect this to fail on platforms that don't have serial port menu. It would be best to run these steps only if `DASHARO_SERIAL_PORT_MENU_SUPPORT` is set.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/pull/601/commits/df81f204ccb46eec1845c46fef3e36e99722ebb7

At this moment every platform supports it. It's defined as ${TRUE} in the `default.robot` and never overwritten. But that's a good point. Thanks.